### PR TITLE
chore: add code of conduct and contribution guidelines file

### LINKED ISSUE #34: Implement recommended community standards to the project
Hello @DenverCoder1 

I was going though this project and noticed that there are some missing items based on the [Github Community Standards recommendations](https://github.com/DenverCoder1/github-readme-youtube-cards/community) that would make the project contributor friendly, e.g. a code of conduct, a contributing guide file, an issue template and a pull request template.

Let me know if this is something you would like to implement on this project.

### PR DESCRIPTION
**Pull request by orama254**
## Type of change

- [ ] Bug fix (added a non-breaking change which fixes an issue)
- [ ] New feature (added a non-breaking change which adds functionality)
- [x] Updated documentation (updated the readme, templates, or other repo files)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

- [ ] Added or updated test cases to test new features

For the readme file, I have made a refactor based on the [custom Icon Badges](https://github.com/DenverCoder1/custom-icon-badges) project to avoid repetition.

fixes #34

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+jonah@freshidea.com.
```
Comment (orama254) [inline CODE_OF_CONDUCT.md:63]: also wanted to know if it was ok including this email on the file.

File: README.md [hunk 2]
```diff
@@ -111,37 +111,7 @@ Contributions are welcome! Feel free to open an issue or submit a pull request i
...
-python action.py --channel_id=UCipSxT7a3rn81vGLw9lqRkg
-```
-
-### Running tests
-
-```bash
-tox
-```
-
+Please check out our [contributing guidelines](/CONTRIBUTING.md) for more information on how to contribute to this project.
```
Comment (DenverCoder1) [inline README.md:114]: It's good for consistency to have an empty line before the heading

```suggestion
Please check out our [contributing guidelines](/CONTRIBUTING.md) for more information on how to contribute to this project.

```

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

I only have one small suggestion

File: README.md (hunk 2 above)
Comment (orama254) [reply README.md:114]: @DenverCoder1 I've pushed a new commit with the requested changes.

**Comment by DenverCoder1** [comment]
Great! :+1: Thanks for your help.

**Pull request merged**